device/project.godot

```diff
@@ -32,6 +32,8 @@ application/save_data="res://globals/save_data.gd"
 application/speech_suffix=".spx"
 application/tooltip_lang_default="en"
 application/speech_path="res://demo/audio/speech/"
+platform/skip_cache=false
+platform/path_to_cache_list="res://game/scenes_cache.gd"
```

Comment:
Shouldn't the default path be `res://game/scenes_cache.gd`, and a placeholder file exist in that location?

Reply:
As said on IRC, the `path_to_cache_list` was a failed experiment that ended up in the PR by accident. I was trying to make it configurable, so the game developer wouldn't have to commit into the framework code to change what gets cached.

I think this is related to bug godotengine/godot#6798 - not sure I interpret it correctly - but anyway the setting cannot be used for `preload()` for whatever reason.

I'll remove the setting and postpone worrying about separation of concerns into the future.